classes/class-kco-templates.php

```diff
@@ -150,14 +150,24 @@ public function check_that_kco_template_has_loaded() {
 				return;
 			}
 
-			$url = add_query_arg(
-				array(
-					'kco-order' => 'error',
-					'reason'    => base64_encode( __( 'Failed to load Klarna Checkout template file.', 'klarna-checkout-for-woocommerce' ) ), // phpcs:ignore WordPress.PHP.DiscouragedPHPFunctions
-				),
-				wc_get_cart_url()
-			);
-			wp_safe_redirect( $url );
+			wc_add_notice( __( 'Klarna Checkout is not available. Please choose a different payment option or contact the store for assistance.', 'klarna-checkout-for-woocommerce' ), 'error' );
```

Comment:
We should not output anything if Klarna can not be shown. This will be handled by WooCommerce if no payment method can be shown.

Id say rather we should print something that is hidden for the customers, for example echo a HTML comment that states "Failed to load Klarna Checkout, for more information look in the plugin logs". Can you try that and see how it looks? Basically its for us to be able to debug issues like this easier when we get support tickets

Reply:
This was added for situations where KCO is the only payment gateway when WooCommerce will still recognize it as `is_available`.

![image](https://github.com/krokedil/klarna-checkout-for-woocommerce/assets/26507935/0517e958-54be-480b-9ebb-b0c365887c4c)